sabnzbd/skintext.py

```diff
@@ -671,6 +671,16 @@
     "explain-pushbullet_apikey": TT("Your personal Pushbullet API key (required)"),  #: Pushbullet settings
     "opt-pushbullet_device": TT("Device"),  #: Pushbullet settings
     "explain-pushbullet_device": TT("Device to which message should be sent"),  #: Pushbullet settings
+    "section-Apprise": TT("Apprise"),  #: Header for Apprise notification section
+    "opt-apprise_enable": TT("Enable Apprise notifications"),  #: Apprise settings
+    "explain-apprise_enable": TT("Send notifications using Apprise URLs"),  #: Apprise settings
+    "opt-apprise_urls": TT("Default Apprise URLs"),  #: Apprise settings
+    "explain-apprise_urls": TT("Use a comma and/or space to identify more then one URL"),  #: Apprise settings
+    "explain-apprise_extra_urls": TT(
+        "Optionally override the default URL(s) for specific cases. To disable a notification for a specific category:"
+        " simply enable it below, but do not provide it a URL to trigger off of. Alternatively, use comma and/or "
```

Comment:
I don't understand this. What can users do here? And how is it different from unchecking a notification-type?

Reply:
It's the same thing. You can drop this if you like. Was just trying to make it clear that anything specified in the individual entries overrides the default (does not stack with it)